### PR TITLE
Windows CI

### PR DESCRIPTION
**Pull request by f-tischler**
- add Windows CI using Github action
- update readme
- remove benchmark from testsuite

#94

### CODE REVIEW

**Comment by f-tischler** [comment]
I think we could easily integrate the Linux build here too. Should I add that or do you prefer to keep the Linux CI on Travis? @jonathan-beard

**Comment by jonathan-beard** [comment]
> I think we could easily integrate the Linux build here too. Should I add that or do you prefer to keep the Linux CI on Travis? @jonathan-beard

bitte sehr! keeping everything in one place seems a much easier way to maintain it.

**Comment by f-tischler** [comment]
Added both Linux and MacOS jobs. 

However, @jonathan-beard you gotta check affinity lib again, does not build on MacOS as of now (some problem with a header not being found in the simple.cpp test).

**Comment by f-tischler** [comment]
Unfortunately, I have not found a way to create a status badge for a single job, thus, there is only one badge for everything instead of a badge per platform

**Comment by jonathan-beard** [comment]
> Added both Linux and MacOS jobs.
> 
> However, @jonathan-beard you gotta check affinity lib again, does not build on MacOS as of now (some problem with a header not being found in the simple.cpp test).

bummer. I wonder if I have header files lurking somewhere in my path that allowed it to build. will check.

File: README.md [hunk 1]
```diff
@@ -6,10 +6,7 @@ User Group / Mailing List: [slack channel](https://join.slack.com/t/raftlib/shar
 
 ### Build status
 
-Platform | Status |
----------| -------|
-Linux | [![Build Status](https://travis-ci.org/RaftLib/RaftLib.svg?branch=master)](https://travis-ci.org/RaftLib/RaftLib) 
-Windows | ![CI](https://github.com/f-tischler/RaftLib/workflows/CI/badge.svg?event=push)
+![CI](https://github.com/f-tischler/RaftLib/workflows/CI/badge.svg?event=push)
```
Comment (jonathan-beard) [inline README.md:9]: Hmm, wondering if these should be set up under the RaftLib org....will it still work with these pointing to your account?

Comment (f-tischler) [reply README.md:9]: Good catch. Github actions are stored in the repository, thus, it should be possible to just edit the link above to point to the original RaftLib repo. Another way would be to merge the PR and then use the context menu in the actions tab to create a new badge link.

Comment (f-tischler) [reply README.md:9]:
```suggestion
![CI](https://github.com/RaftLib/RaftLib/workflows/CI/badge.svg?event=push)
```

**Comment by jonathan-beard** [comment]
Gonna merge then we’ll fix the affinity lib issues for OS X, I think I’m gonna have to find a clean install to test it on.

**Pull request merged**